README.md

```diff
@@ -78,122 +93,218 @@ neolink --config my_config.toml
+`tls_client_auth = "none|request|require"`; in this case the client should
+present a certificate signed by the server's CA.
 
 TLS is disabled by default.
 
-You can password-protect the Neolink server by adding `[[users]]`` sections to the configuration file:
+You can password-protect the Neolink server by adding `[[users]]` sections to
+the configuration file:
+
 ```
 [[users]]
 name: someone
 pass: somepass
 ```
-you also need to add the allowed users into each camera by adding the following to `[[cameras]]`.
+
+you also need to add the allowed users into each camera by adding the following
+to `[[cameras]]`.
+
 ```
 permitted_users = ["someone", "someoneelse"]
 ```
 
-Anywhere a username is accepted it can take any username or one of the following special values.
+Anywhere a username is accepted it can take any username or one of the
+following special values.
+
 - `anyone` means any user with a valid user/pass
 - `anonymous` means no user/pass required
 
 The default `permitted_users` list is:
-- `[ "anonymous"]` if no `[[users]]` were given in the config meaning no authentication required to connect.
-- `[ "anyone" ]` if `[[users]]` were provided meaning any authourised users can connect.
 
-You can change the Neolink log level by setting the `RUST_LOG` environment variable (not in the configuration file) to one of `error`, `warn`, `info`, `debug`, or `trace`:
+- `[ "anonymous"]` if no `[[users]]` were given in the config meaning no
+authentication required to connect.
 
-```
+- `[ "anyone" ]` if `[[users]]` were provided meaning any authourised users can
+connect.
+
+You can change the Neolink log level by setting the `RUST_LOG` environment
+variable (not in the configuration file) to one of `error`, `warn`, `info`,
+`debug`, or `trace`:
+
+```sh
 set RUST_LOG=debug
 ```
 
 On Linux:
 
-```
+```bash
 export RUST_LOG=debug
 ```
 
 ## Viewing
 
-Connect your RTSP client to the stream with the name you provided in the configuration file.
-Again, the default URL is `rtsp://127.0.0.1:8554/your_camera_name` if you're running it on the same computer as the client.
+Connect your RTSP client to the stream with the name you provided in the
+configuration file.
+
+Again, the default URL is `rtsp://127.0.0.1:8554/your_camera_name` if you're
+running it on the same computer as the client.
 The smaller SD video is `rtsp://127.0.0.1:8554/your_camera_name/subStream`.
 
-4K cameras send large video "key frames" once every few seconds and the client must have a receive buffer large enough to store the entire frame.
-If your client's buffer size is configurable (like Blue Iris), ensure it's set to 20MB, which should ensure plenty of headroom.
+4K cameras send large video "key frames" once every few seconds and the client
+must have a receive buffer large enough to store the entire frame.
+If your client's buffer size is configurable (like Blue Iris), ensure it's set
+to 20MB, which should ensure plenty of headroom.
+
+## Motion Detection and MQTT
```

Comment:
I don't see any MQTT implementation in this PR (probably okay to leave it out). Could you remove this readme section?

Reply:
Oh yes I didn't realise this snuck in